src/quiz/Quiz.js

```diff
@@ -65,40 +49,69 @@ export default class Quiz extends Component {
         summary={this.props.results[resultNumber].summery.summery} />
     }
   }
+  getAnswerMatchResult() {
+    const totalQuestions = this.props.questions.length;
+    const totalAnswered = this.state.userAnswers.length;
+    const counts = {};
+    const compare = -1;
+    const mostFrequent = [];
+    for (i = 0; i <= this.totalAnswered.length; i++) {
+      const resultAnswer = this.userAnswer.answerAnswerMatch[i];
+      if (counts[resultAnswer] === undefined) {
+        counts[resultAnswer] = 1;
+      } else {
+        counts[resultAnswer] = counts[resultAnswer] + 1;
```

Comment:
The way you use counts to determine the most frequent answer is great. That's how I would do it.

I get confused when you do stuff with "answerAnswerMatch". What is that? Also, should we do any of this computation unless we know they've answered all the questions? Looks like we're doing it each time.